renderer/src/components/ActivityChartCard.jsx

```
@@ -24,12 +35,29 @@ function ActivityChartCard({ title, description, chartType }) {
         if (!response.ok) {
           throw new Error(`HTTP error! Status: ${response.status}`);
         }
-        return response.json();
+        
+        const contentType = response.headers.get('content-type');
+        if (contentType && contentType.includes('image/png')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)